recipes/r-emstreer/meta.yaml

```
@@ -0,0 +1,102 @@
+{% set version = '3.0.0' %}
+
+{% set posix = 'm2-' if win else '' %}
+{% set native = 'm2w64-' if win else '' %}
+
+package:
+  name: r-emstreer
+  version: {{ version|replace("-", "_") }}
+
+source:
+  url:
+    - {{ cran_mirror }}/src/contrib/emstreeR_{{ version }}.tar.gz
+    - {{ cran_mirror }}/src/contrib/Archive/emstreeR/emstreeR_{{ version }}.tar.gz
+  sha256: 4b56ee8430b350222ee9d38a2b33557cdbc21c52a1e430fc0ce8f56963e2cfe5
+
+build:
+  merge_build_host: True  # [win]
+  # If this is a new build for the same version, increment the build number.
+  number: 0
+  # no skip
+  noarch: generic
+
+  # This is required to make R link correctly on Linux.
+  rpaths:
+    - lib/R/lib/
+    - lib/
+
+requirements:
+  build:
+    - {{ posix }}zip               # [win]
+    - cross-r-base {{ r_base }}    # [build_platform != target_platform]
+
+  host:
+    - r-base
+    - r-bbmisc
+    - r-ggplot2
+    - r-mlpack
+    - r-scatterplot3d
+
+  run:
+    - r-base
+    - r-bbmisc
+    - r-ggplot2
+    - r-mlpack
+    - r-scatterplot3d
+
+test:
+  commands:
+    # You can put additional test commands to be run here.
+    - $R -e "library('emstreeR')"           # [not win]
+    - "\"%R%\" -e \"library('emstreeR')\""  # [win]
+
+  # You can also put a file called run_test.py, run_test.sh, or run_test.bat
+  # in the recipe that will be run at test time.
+
+  # requires:
+    # Put any additional test requirements here.
+
+about:
+  home: https://CRAN.R-project.org/package=emstreeR
+  license: BSD_3_clause
```

You can lookup the short spdx identifier here https://spdx.org/licenses/BSD-3-Clause.html
```suggestion
  license: BSD-3-Clause
```